sensors/shtc3Sensor/SHTC3Sensor.cpp

```
@@ -0,0 +1,146 @@
+}
+
+bool SHTC3Sensor::wakeUp() {
+    return (shtc1_wake_up() == STATUS_OK);
+}
+
+SHTC3Sensor::~SHTC3Sensor() {
+    stopContinuousMeasurement();
+}
+
+void SHTC3Sensor::setMeasurementInterval(uint32_t interval_ms) {
+    if (interval_ms == 0 || interval_ms < kMinMeasurementIntervalMs) {
```

Clang (14.0.6)

[warning] if with identical then and else branches

(bugprone-branch-clone)